internal/index/indexscanner/testdata/testindex/receipts/custom.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-plugin
+spec:
+  platforms:
+  - bin: kubectl-test-plugin
+    files:
+    - from: ./*.sh
+      to: .
+    selector:
+      matchLabels:
+        arch: amd64
+        os: linux
+    sha256: deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef
+    uri: http://example.com/
+  shortDescription: in-memory test plugin object
+  version: v1.0.0-test.1
+Status:
```

Comment:
This shouldnt be parsing, right?

Reply:
I didn't actually notice that I capitalized those. Let me look into it.

Reply:
I thought the json struct tags would affect how it was parsed but it doesn't seem to be that way. When the receipt is written it uses lower case letters for those keys. Am I missing something?

Reply:
This just shouldn’t be parsing accurately.
We currently ignore unknown fields intentionally.

Reply:
I'm a little confused I think. What do you mean by unknown fields in this case? If you're referring specifically to the test receipt that has a custom index set in its status, I mainly added that even though only default can be set in a receipt because `receipt.Load` doesn't validate the index that is parsed. Let me know if I misunderstood.

Reply:
All our fields are case-sensitive.
So as you can imagine, there can't be a field capitalized like `Status`.
So typing `Status` here should be the same as `Asadklfjdksfj` (which would be ignored).

Reply:
I tried changing `version` to `Version` in my receipts in my normal krew installation and was still able to parse receipts in `kubectl krew list` (where it loads the receipts and gets `.Spec.Version` from them). Maybe I'm doing something wrong. I also thought they were case sensitive and made a mistake by accidentally making those capitalized in the test file, but didn't notice it since the tests were passing and they were still parsing correctly.

Reply:
That's very odd. Let's continue thinking they're case-sensitive.